src/containers/BooksForm.js

```diff
@@ -17,10 +48,23 @@ function BooksForm() {
         }
       </select>
 
-      <button type="submit">Submit</button>
-
+      <button type="submit" onClick={handleSubmit}>Submit</button>
+      <p>
+        hello:
+        {title}
+      </p>
```

Comment:
- [ ] It seems that this p tag was implemented for testing purposes. We can safely remove it by now. Let's remove it before merging the branch.

Reply:
Thanks for your review and approval. I will definitely remove that p tag, it was just for testing purposes.